### PR TITLE
INV-153 Added supplier to the inventory item collection GET filter

### PR DESCRIPTION
**Pull request by jennshevach**
Updated filter for GET endpoint on the inventory item collection by adding supplier and renaming manufacturer part numbers.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -70,15 +70,17 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
 This endpoint allows for the retrieval of a list of inventory items as well as the creation of new ones
 
 + Parameters
     + accountId: 2 (required, number) - {INT} ID of the account in the form of an integer
     + q: `P/N 234234A` (string, optional) - Custom query object. Searches name and description fields.
     + isActive: true (boolean, optional) - Filter inventory items by active or inactive status.
     + manufacturerIds: 1 (array[number], optional) - {INT} Filter inventory items using the ID of the manufacturer as a integer.
-    + partNumbers: `RES-100-0003` (array[string], optional) - Filter inventory items using the part number of the item, specific to manufacturer.
+    + manufacturerPartNumbers: `RES-100-0003` (array[string], optional) - Filter inventory items using the part number of the item, specific to manufacturer. Partial matches are allowed.
+    + supplierIds: 1 (array[number], optional) - {INT} Filter inventory items using the ID of the supplier as a integer.
```
Comment (ghost) [inline apiary.apib]: Grammatical error - "...as a integer"  should be "as an integer"

**Review by ghost — COMMENTED** [review]
Looks good other than the grammar issue

File: apiary.apib [hunk 2]
```diff
@@ -70,15 +70,17 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
 
-## Inventory Item Collection [/accounts/{accountId}/inventoryitems{?q,isActive,manufacturerIds,partNumbers,page,pagesize,orderby}]
+## Inventory Item Collection [/accounts/{accountId}/inventoryitems{?q,isActive,manufacturerIds,manufacturerPartNumbers,supplierIds,supplierPartNumbers,page,pagesize,orderby}]
 This endpoint allows for the retrieval of a list of inventory items as well as the creation of new ones
 
 + Parameters
     + accountId: 2 (required, number) - {INT} ID of the account in the form of an integer
     + q: `P/N 234234A` (string, optional) - Custom query object. Searches name and description fields.
     + isActive: true (boolean, optional) - Filter inventory items by active or inactive status.
     + manufacturerIds: 1 (array[number], optional) - {INT} Filter inventory items using the ID of the manufacturer as a integer.
```
Comment (ghost) [inline apiary.apib]: Grammatical error - "...as a integer"  should be "as an integer"

**Review by ghost — APPROVED** [review]

**Review by jbeef710 — COMMENTED** [review]
Looks good

**Review by jbeef710 — APPROVED** [review]

**Pull request merged**